<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="TabWidget" name="consoleTabulator">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <property name="Title" stdset="0">
     <string/>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Actions">
    <property name="title">
     <string>&amp;Actions</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actAbout"/>
    <addaction name="actAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Actions"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actAboutQt">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabWidget</class>
   <extends>QTabWidget</extends>
   <header>tabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
